integration-tests/utils/integration-config.js

```diff
@@ -3,7 +3,7 @@ const {
   TERRA_TOKEN: bearerToken,
   BILLING_PROJECT: billingProject = 'general-dev-billing-account',
   WORKFLOW_NAME: workflowName = 'haplotypecaller-gvcf-gatk4',
-  SCREENSHOT_DIR: screenshotDir = 'failure-screenshots'
```

Comment:
TOL: I'm ever so slightly in favor of favoring capturing screenshots by default. If we assume that the tests are pretty stable, then we'll most often see failures in edge cases that we might not be able to readily reproduce after opting-in.

Reply:
Let's revisit this in a followup PR. It's a fair point, I just want to make sure we get the management of the files right.